docs/api/model_selection/cross_validation.rst

```diff
@@ -70,13 +72,16 @@ In this next example we show how to visualize cross-validated scores for a regre
     from sklearn.model_selection import KFold
 
 
-    energy = load_data("energy")
+    # Load the regression data set
```

Comment:
I changed the variable name `energy` to `data` just to be consistent with the rest of the documentation.